pkg/webhook/admission/pod/storage_initializer_injector.go

```diff
@@ -66,6 +66,7 @@ type StorageInitializerConfig struct {
 	EnableDirectPvcVolumeMount bool   `json:"enableDirectPvcVolumeMount"`
 	EnableOciImageSource       bool   `json:"enableModelcar"`
 	UidModelcar                *int64 `json:"uidModelcar"`
+	ModelCachePvcName          string `json:"modelCachePvcName"`
```

Comment:
If we go with the pvc per model cache CR approach then we do not need this configuration.